scipy/ndimage/tests/test_filters.py

```diff
@@ -190,8 +189,7 @@ def test_correlate01(self, xp):
         assert_array_almost_equal(output, expected)
 
     @xfail_xp_backends('cupy', reason="Differs by a factor of two?")
-    @skip_xp_backends("jax.numpy", reasons=["output array is read-only."],
-                      cpu_only=True, exceptions=['cupy', 'jax.numpy'])
+    @skip_xp_backends("jax.numpy", reasons=["output array is read-only."])
     def test_correlate01_overlap(self, xp):
```

Comment:
I think there is a regression here? PyTorch GPU would be skipped before but not skipped now.

Reply:
Ah, yes, I see. 3025b7f7d5765abadf8f97cc1ed00a9bad848e4d and afe9d96b79b5d151abe10fae047d971b9497abd6 ?

Not having a CI makes it a bit inconvenient, any chance you're set up to run the full suite on all backends, including GPU ones?